src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -42,11 +42,18 @@ MAY be included alongside the MEG data; examples are provided below.
 This template is for MEG data of any kind, including but not limited to
 task-based, resting-state, and noise recordings. If multiple Tasks were
 performed within a single Run, the task description can be set to
-`task-multitask`. The \_meg.json SHOULD contain details on the Tasks. Some
-manufacturers data storage conventions use folders which contain data files of
-various nature: e.g., CTF’s .ds format, or BTi/4D. Please refer to
-[Appendix VI](../99-appendices/06-meg-file-formats.md) for examples from a
-selection of MEG manufacturers.
+`task-multitask`. The \_meg.json SHOULD contain details on the Tasks.
+
+Some manufacturers data storage conventions use folders which contain data
+files of various nature: e.g., CTF's `.ds` format, or BTi/4D.
```

Comment:
Plural possessive. Also using `for example` rather than `e.g.` for screen readers.

```suggestion
Some manufacturers' data storage conventions use folders which contain data
files of various nature: for example, CTF's `.ds` format, or BTi/4D.
```